<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedingWidget</class>
 <widget class="QWidget" name="FeedingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>喂料管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="child_widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_min_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_max_restore_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="feedstore_cose_btn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 78, 111);</string>
        </property>
        <widget class="NavButton" name="feeding_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>喂料</string>
         </property>
        </widget>
        <widget class="NavButton" name="feeding_detail_opt">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>140</width>
           <height>35</height>
          </rect>
         </property>
         <property name="text">
          <string>喂料明细</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(41, 57, 86);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="feeding_stackedWidget">
        <property name="minimumSize">
         <size>
          <width>768</width>
          <height>581</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 78, 111);</string>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <widget class="QPushButton" name="area_feed_search_btn">
             <property name="geometry">
              <rect>
               <x>310</x>
               <y>20</y>
               <width>80</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLineEdit" name="area_feed_edit">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>20</y>
               <width>113</width>
               <height>28</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>71</width>
               <height>28</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>物品名称：</string>
             </property>
            </widget>
            <widget class="QPushButton" name="feeding_btn">
             <property name="geometry">
              <rect>
               <x>400</x>
               <y>20</y>
               <width>80</width>
               <height>28</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QComboBox" name="area3_combox">
             <property name="geometry">
              <rect>
               <x>220</x>
               <y>20</y>
               <width>80</width>
               <height>28</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(171, 171, 171);
border-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="area_feedstore_tableView"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>开始时间：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="startdateEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(171, 171, 171);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>结束时间：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="enddateEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>110</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(171, 171, 171);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="area_combox">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(171, 171, 171);
border-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="area_item_combox">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>28</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(171, 171, 171);
border-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="detail_search_btn">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>28</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>28</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="feeding_detail_tableView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavButton</class>
   <extends>QPushButton</extends>
   <header location="global">qnavbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
